docker/docker-compose.ci.yml

```diff
@@ -26,6 +26,7 @@ services:
     working_dir: /home/unlock
     expose:
       - 3000
+      - 3001
```

Comment:
Does it mean that the unlock image exposes 2 ports? I am not sure this is right since there is only one service running?

Reply:
the split-off paywall runs on 3001. This allows accessing it within unlock (currently not possible)